contributors/design-proposals/scheduling/scheduling-policy.md

```diff
@@ -195,66 +332,355 @@ If `allowedAffinities` is totally absent from the spec, no affinity is allowed w
-    - preferredDuringSchedulingIgnoredDuringExecution
+  required:
+    affinities:
+      nodeAffinities:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - keys: ["beta.kubernetes.io/arch"]
+              operators: ["In"]
+              values: ["amd64", "arm64"]
+  allowed:
+    affinities:
+      nodeAffinities:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - keys: ["failure-domain.beta.kubernetes.io/region","kubernetes.io/authorized-region"]
+              operators: ["In","NotIn"]
+              values: ["eu-2", "us-1"]
+      podAntiAffinities: {}
+  default:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - keys: ["beta.kubernetes.io/arch"]
+              operators: ["In"]
+              values: ["amd64"]
+```
+
+In this example, we allow:
+- hard NodeAffinity based on
+  -  `beta.kubernetes.io/arch` if value is `amd64` or `arm64`. Defaults to `amd64` if unspecified.
+  -  (optionally) any combination of <keys, operators, values> specified in `allowed` secion.
+- All podAntiAffinities
+- No podAffinities
+
+```yaml
+Kind: SchedulingPolicy
+spec:
+  allowed:
+    affinities:
+      nodeAffinities:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - keys: ["failure-domain.beta.kubernetes.io/region","kubernetes.io/authorized-region"]
+              operators: ["In","NotIn"]
+              values: ["eu-2", "us-1"]
+          - matchExpressions:
+            - keys: ["failure-domain.beta.kubernetes.io/zone"]
+              operators: ["NotIn"]
+              values: ["dc1", "dc2"]
+          podAntiAffinities: {}
 ```
 
+This example highlights the case where you don't want a full combinatory, here we allow the same affinities as the previous example, in addition,
+we allow a match expression that checks that some zone labels are not in the specified values.
+
+
 ##### Allow-all policy
 In this example, all affinities are allowed:
+
 ```yaml
 Kind: SchedulingPolicy
 spec:
-  allowedAffinities:
-    nodeAffinities: []
-    podAffinities: []
-    podAntiAffinities: []
+  allowed:
+    affinities: {}
+```
+
+Which is equivalent to:
+
+```yaml
+Kind: SchedulingPolicy
+spec:
+  allowed:
+    affinities:
+      nodeAffinities: {}
+      podAffinities: {}
+      podAntiAffinities: {}
 ```
 
 If a sub-item of allowedAffinities is absent from SchedulingPolicy, it is not allowed e.g:
+
 ```yaml
 Kind: SchedulingPolicy
 spec:
   allowedAffinities:
-    nodeAffinities: []
+    nodeAffinities: {}
+```
+
+In this example, only nodeAffinities (required and preferred) are allowed but no podAffinities nor podAntiAffinities.
+
+## Multiple SchedulingPolicies considerations
```

Comment:
@liggitt @bsalamat @tallclair @davidopp - these are the option available to handle conflicts.

concerns for me:
- Option 1 is way too complicated to handle. 
- Option 2 is more deterministic but won't let users split `SchedulingPolicy` accross multiple resources.

Reply:
why we have to merge them? Does go through the policy one by one not enough?

Reply:
@k82cn - it is not a proper "merge", i.e we do not aggregate `SchedulingPolicy` resources into a new one. we'll be going through policies and resolves conflicts to compute the set of rules, these options are the possible ways of dealing with such conflicts.

Reply:
Great